### ISSUE TITLE
We should support tokens

### ISSUE THREAD

**Issue by silasary** [PennyDreadfulMTG/Penny-Dreadful-Tools#189]
When discussing plays or combos, people sometimes mention [Saproling Token]s or [Eldrazi Spawn Token]s.

We should probably display them.

**Comment by bakert** [comment]
I think it's pretty clear we don't want them in !search but supporting them in square brackets makes sense to me. I think we just need to not filter out layout='token' as we currently do.

**Comment by bakert** [comment]
I tried this and it "worked":

```
diff --git a/command.py b/command.py
index 542a06c..5fb635b 100644
--- a/command.py
+++ b/command.py
@@ -299,7 +299,7 @@ def cards_from_query(query):
             query = canonicalize(name)

     cards = oracle.search(query)
-    cards = [card for card in cards if card.layout != 'token' and card.type != 'Vanguard']
+    cards = [card for card in cards if card.type != 'Vanguard']

     # First look for an exact match.
     for card in cards:
```

The problem being that bluebones.net doesn't currently do token images so something like [Elf] which finds Elf Warrior and Elf Replica shows only the replica. No compositing.

I could go either way on this one. It might make sense to exclude things like Planechase, too, and just say we only do cards. Or maybe we should get bluebones to start supporting tokens and planechase and Vanguard and Schemes. I'd be pretty happy to close this wontfix though.

**Comment by bakert** [comment]
Should we support tokens? I'm ok to close this one wontfix.

**Comment by bakert** [comment]
Closing wontfix for now. We can always revisit.

**Comment by silasary** [comment]
I'd say it's blocked on doing local compositing.
